twitter_clone_app/settings.py

```diff
@@ -134,3 +134,7 @@
 # https://docs.djangoproject.com/en/3.2/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+# LOGIN REDIRECT URL
+LOGIN_REDIRECT_URL = '/home'
```

Comment:
テストなどで名前から逆引きして値を渡してあげる書き方にしているので統一することをお勧めします。
```suggestion
LOGIN_REDIRECT_URL = 'user:home'
```

Reply:
修正しました